src/app/clusters/bindings/bindings.cpp

```diff
@@ -150,6 +173,16 @@ bool emberAfBindingClusterUnbindCallback(app::CommandHandler * commandObj, const
         return true;
     }
 
+    if (GetNumberOfUnicastBindingForNode(fabricIndex, nodeId) == 1)
```

Comment:
I'd suggest that this code should:
1. Only be run after emberDeleteBinding (though I don't feel too strongly about this).
2. Only run when we are removing a unicast binding (this I do feel strongly about; using nodeId if we are a group binding removal makes no sense).